src/DGtal/dec/doc/moduleDECEmbedding.dox

```diff
@@ -0,0 +1,130 @@
+/**
+ * @file moduleDECEmbedding.dox
+ * @author David Coeurjolly (\c david.coeurjolly@liris.cnrs.fr )
+ * @author Pierre Gueth (\c pierre.gueth@liris.cnrs.fr )
+ * Laboratoire d'InfoRmatique en Image et Systemes d'information - LIRIS (CNRS, UMR 5205), CNRS, France
+ *
+ * @date 2014/04/04
+ *
+ * Documentation file for feature DECEmbedding
+ *
+ * This file is part of the DGtal library.
+ */
+
+/*
+ * Useful to avoid writing DGtal:: in front of every class.
+ * Do not forget to add an entry in src/DGtal/base/Config.h.in !
+ */
+namespace DGtal {
+//----------------------------------------
+/*!
+@page moduleDECEmbedding Embedding
+@writers Pierre Gueth
+
+[TOC]
+
+\section sectDECEmbedding Embedding
```

Comment:
this is used as a ref elsewhere so i think it would make more sense to keep a more specific term instead of introduction.